src/main/java/com/google/sps/servlets/getVideosServlet.java

```diff
@@ -66,6 +66,16 @@
   private static final Logger logger = Logger.getLogger(getVideosServlet.class.getName());
   private static final String KIND = new String("Video");
 
+  public static HashMap<String, String> getVideoStringFromAPI (String youtubeapiKey, ArrayList<String> videoCategoryNames, HashMap<String, String> videoMap, HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Do you need this method to be public? If no other class is using this method, you can just make it private.

Reply:
Ok -- I'll change that now. :)